popmon/visualization/templates/section.html

```diff
@@ -55,6 +55,10 @@ <h3> {{ feature.titles.get(ref, ref) }} </h3>
     {%- else -%}
         <div class="row" >
             {%- for metric in plots -%}
+              <script> 
+                {%- set curr = loop.index  -%}
```

Comment:
Table is now aligned to the left.
This can be solved (al least in safari) by putting `.js-plotly-plot{margin:auto;}` in the css file.